src/main/java/net/snowflake/ingest/streaming/internal/RowBufferStats.java

```diff
@@ -306,20 +306,22 @@ byte[] performConversion(String input) {
   private long currentMaxLength;
   private CollationDefinition collationDefinition;
   private final String collationDefinitionString;
+  private final String columnDisplayName;
```

Comment:
could you add a comment explain why we need it here?

Reply:
Comment added